ravenframework/CodeInterfaceClasses/MELCOR/melcorCombinedInterface.py

```diff
@@ -13,104 +13,168 @@
 # limitations under the License.
 """
   Created on April 18, 2017
-  @author: Matteo Donorio (University of Rome La Sapienza),
-           Fabio Gianneti (University of Rome La Sapienza),
+  @author: Matteo D'Onorio (Sapienza University of Rome)
            Andrea Alfonsi (INL)
 """
+
+import os
+from ...contrib.melcorTools import melcorTools
 from ravenframework.CodeInterfaceBaseClass import CodeInterfaceBase
-from .melcorInterface   import MelcorApp
-from .melgenInterface import MelgenApp
+from ..Generic import GenericParser
+import pandas as pd
+
 
 class Melcor(CodeInterfaceBase):
```

Comment:
Can we make Melcor interface inherit from GenericCodeInterface? I saw you are using GenericParser already.